products/permissions.py

```diff
@@ -0,0 +1,28 @@
+"""
+Contains custom permissions for products app
+"""
+from rest_framework.permissions import BasePermission
+
+from users.contants import CONTENT_MANAGER
+
+
+class IsContentManager(BasePermission):
+    """
+    Tells if user can create or update products
+    """
+
+    def has_permission(self, request, view):
+        """
+        True to grant permission
+        False to deny permission
+        Args:
+            request():
+            view():
```

Comment:
You might want to mention the Base class names here not the method representation.

Reply:
Done